Lib/test/test_pathlib.py

```diff
@@ -2478,6 +2478,207 @@ def test_complex_symlinks_relative(self):
     def test_complex_symlinks_relative_dot_dot(self):
         self._check_complex_symlinks(os.path.join('dirA', '..'))
 
+class WalkTests(unittest.TestCase):
+    cls = pathlib.Path
```

Comment:
I'm afraid I wasn't clear enough about what I was asking for previously. It wasn't so much to not assign `pathlib.Path` to `P`, but to not have a variable named `P`. It's too short, easy to overlook when skimming the code, and isn't descriptive on its own.

```suggestion
```